volontulo/templates/common/col1.html

```diff
@@ -1,13 +1,21 @@
 {% extends "common/base.html" %}
 
 {% block layout %}
-
-    <div class="row">
-        <div class="col-xs-12">
-        {% block content %}
-        {% endblock %}
+<section class="content-heading">
```

Comment:
yeah, there would be a need for some `if` condition, right? Couldn't figure it out
In html it is no problem, because it doesn't take any vertical space at this point